[2022-04-19 13:03:32,474] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: hook_dag.log_end_no_data_1 manual__2022-04-19T13:03:21.233989+00:00 [queued]>
[2022-04-19 13:03:32,485] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: hook_dag.log_end_no_data_1 manual__2022-04-19T13:03:21.233989+00:00 [queued]>
[2022-04-19 13:03:32,485] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-04-19 13:03:32,485] {taskinstance.py:1244} INFO - Starting attempt 1 of 2
[2022-04-19 13:03:32,485] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-04-19 13:03:32,494] {taskinstance.py:1264} INFO - Executing <Task(PostgresOperator): log_end_no_data_1> on 2022-04-19 13:03:21.233989+00:00
[2022-04-19 13:03:32,499] {standard_task_runner.py:52} INFO - Started process 12656 to run task
[2022-04-19 13:03:32,501] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'hook_dag', 'log_end_no_data_1', 'manual__2022-04-19T13:03:21.233989+00:00', '--job-id', '390', '--raw', '--subdir', 'DAGS_FOLDER/hook_dag.py', '--cfg-path', '/tmp/tmpu5b4y67v', '--error-file', '/tmp/tmphzm_ugiu']
[2022-04-19 13:03:32,502] {standard_task_runner.py:77} INFO - Job 390: Subtask log_end_no_data_1
[2022-04-19 13:03:32,565] {logging_mixin.py:109} INFO - Running <TaskInstance: hook_dag.log_end_no_data_1 manual__2022-04-19T13:03:21.233989+00:00 [running]> on host 22e04adb6bfa
[2022-04-19 13:03:32,613] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=hook_dag
AIRFLOW_CTX_TASK_ID=log_end_no_data_1
AIRFLOW_CTX_EXECUTION_DATE=2022-04-19T13:03:21.233989+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-04-19T13:03:21.233989+00:00
[2022-04-19 13:03:32,618] {base.py:79} INFO - Using connection to: id: postgres_default. Host: postgres, Port: 5432, Schema: ***, Login: ***, Password: ***, extra: {}
[2022-04-19 13:03:32,622] {dbapi.py:225} INFO - Running statement: UPDATE list_flow_log
SET q_source=null, q_target=null, t_end=NOW(), status=2, optional_key=null, comment = 'not_new_data'
where id = 13;
, parameters: None
[2022-04-19 13:03:32,625] {dbapi.py:233} INFO - Rows affected: 1
[2022-04-19 13:03:32,635] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=hook_dag, task_id=log_end_no_data_1, execution_date=20220419T130321, start_date=20220419T130332, end_date=20220419T130332
[2022-04-19 13:03:32,676] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-19 13:03:32,698] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
